pkgs/tools/compression/lzip/mingw-install-exe-file.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile.in b/Makefile.in
+index d07ad5a..1c15203 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -64,7 +64,7 @@ install-strip-compress : install-bin-strip install-info-compress install-man-com
+ 
+ install-bin : all
+ 	if [ ! -d "$(DESTDIR)$(bindir)" ] ; then $(INSTALL_DIR) "$(DESTDIR)$(bindir)" ; fi
+-	$(INSTALL_PROGRAM) ./$(progname) "$(DESTDIR)$(bindir)/$(progname)"
++	$(INSTALL_PROGRAM) ./$(progname).exe "$(DESTDIR)$(bindir)/$(progname).exe"
```

Comment:
```suggestion
+	$(INSTALL_PROGRAM) ./$(progname)$(EXEEXT) "$(DESTDIR)$(bindir)/$(progname)$(EXEEXT)"
```

https://www.gnu.org/software/automake/manual/html_node/EXEEXT.html

Reply:
> However, Automake cannot apply this rewriting to configure substitutions

There's no configure.ac, so the change probably doesn't work.

Reply:
Ok thanks for trying.